### ISSUE TITLE
Embed keyboard shortcut in a tip

### ISSUE THREAD

**Issue by AmadeusW** [justcla/HotTips#23]
Suppose we have a tip that introduces a behavior triggered by a keyboard shortcut, e.g. `Alt PgUp`

We may need to probe the keyboard shortcut module to get the applicable keys in user's current keyboard layout. Then, we take this data and display it in the tip.

Alternatively, we pick a single keyboard layout and use its keys throughout all tips.

**Comment by justcla** [comment]
Right! I was thinking we would show the shortcut in the "(Default)" layout. But thought we might be able to have a button or link to "Change shortcut", which would pop a simple shortcut changing tool (being built by Josh Stevens).

**Comment by justcla** [comment]
Also, I considered that we could check the system to determine what the current shortcut is for a given command, and show it as a note.

ie. Note: You currently have GoToNextIssue configured to shortcut: Ctrl+PgDn
To Change the shortcut, click here.

**Comment by justcla** [comment]
But sometimes a tip shows multiple shortcuts.
:-/
And a user can have several shortcuts (or none) currently hooked up to a command.
The TipInfo would need to contain a mapping of all the commands in the tip, to all their shortcuts. And then we have to work out how to display multiple commands and shortcuts.

**Comment by AmadeusW** [comment]
Is it even possible to fetch the keyboard shortcuts applicable to a given command?
